unittests/test_modules.py

```diff
@@ -402,3 +402,45 @@ def test_mapping_deep_cycle(self):
         }
         self.assertRaisesRegex(EnvironError, 'm0->m1->m3->m4->m2->m1',
                                self.modules_system.load_module, 'm0')
+
+    def test_mapping_from_file_simple(self):
+        with open('module_mappings_file.txt', 'w') as f:
+            print('m1:m2, m3'
+                  '\nm2 : m4',
+                  '\n  #m5: m6',
+                  '\n',
+                  '\nm1: m7, # An inline comment after the coma',
+                  '\nm1: m7 # An inline comment with no coma',
+                  file=f)
```

Comment:
Do not use the `print()` here. Write it as follows:

```python
fp.write('m1:m2, m3\n'
         'm2 : m4\n'
         '  #m5: m6\n'
         '\n'
         'm1: m7, # An inline comment after the comma\n'
         'm1: m7 # An inline comment with no comma\n')
```

_Notice the lack of commas at the end of each line. Python will concatenate the strings._

Also the comment is still safe and sound and never got in a [coma](https://en.wikipedia.org/wiki/Coma). 😂 Make sure to spell it as comma.